src/kedro_databricks/plugin.py

```diff
@@ -117,6 +117,7 @@ def bundle(
 @click.option("-c", "--conf", default=DEFAULT_CONF_FOLDER, help=CONF_HELP)
 @click.option("-d", "--debug/--no-debug", default=False, help="Enable debug mode")
 @click.option("-p", "--pipeline", default=None, help="Bundle a single pipeline")
+@click.option("-v", "--var", default=None, help="set values for variables defined in bundle config. Example: --var=\"foo=bar\"")
```

Comment:
So Databricks allows you to pass this multiple times.... should we do the same?

Reply:
Good question, did not see it was possible.

I would say we should do as they do since we are trying to replicate what is doing databricks cli. What do you think?

Reply:
We should 😊 this was such a minor addition I actually implemented it yesterday when I was doing some or refactoring. See PR #80.

It just needs tests before it can be merged

Reply:
I saw the MR, it s perfect, thank you for that!